projects/plugins/boost/app/assets/src/js/api/speed-scores.ts

```diff
@@ -21,11 +21,20 @@ type SpeedScores = {
 	desktop: number;
 };
 
+type SpeedScoresSet = {
+	current: SpeedScores;
+	previous: SpeedScores;
+};
+
 type ParsedApiResponse = {
 	id?: string;
-	scores?: SpeedScores;
+	scores?: SpeedScoresSet;
 };
 
+function getResponseSpeedScoresSet( response: ParsedApiResponse ): SpeedScoresSet {
```

Comment:
I don't think this getter is necessary, nor improves code readability. The `ParsedApiResponse` type already defines a `scores` member, and this just accesses it indirectly.